resources/i18n/hi_HI/cura.po

```diff
@@ -0,0 +1,7748 @@
+msgctxt "@title:menu menubar:file"
+msgid "Open File(s)..."
+msgstr "फ़ाइलें खोलें..."
+
+
+#: /home/trin/Gedeeld/Projects/Cura/resources/qml/Menus/PrinterMenu.qml:25
+msgctxt "@label:category menu label"
+msgid "Network enabled printers"
+msgstr "नेटवर्क सक्षम प्रिंटर"
+
+
+#: /home/trin/Gedeeld/Projects/Cura/resources/qml/Menus/PrinterMenu.qml:42
+msgctxt "@label:category menu label"
+msgid "Local printers"
+msgstr "स्थानीय प्रिंटर"
+
+
+#: /home/trin/Gedeeld/Projects/Cura/resources/qml/Menus/RecentFilesMenu.qml:15
+msgctxt "@title:menu menubar:file"
+msgid "Open &Recent"
+msgstr "ओपन एंड रिसेट"
+
+
+#: /home/trin/Gedeeld/Projects/Cura/resources/qml/Menus/SaveProjectMenu.qml:15
+msgctxt "@title:menu menubar:file"
+msgid "Save Project..."
+msgstr "प्रोजेक्ट सेव करें..."
+
+
+#: /home/trin/Gedeeld/Projects/Cura/resources/qml/Menus/SettingsMenu.qml:15
+msgctxt "@title:menu menubar:settings"
+msgid "&Printer"
+msgstr "&Printer"
+
+
+#: /home/trin/Gedeeld/Projects/Cura/resources/qml/Menus/SettingsMenu.qml:29
+msgctxt "@title:menu"
+msgid "&Material"
+msgstr "&Materiál"
+
+
+#: /home/trin/Gedeeld/Projects/Cura/resources/qml/Menus/SettingsMenu.qml:44
+msgctxt "@action:inmenu"
+msgid "Set as Active Extruder"
+msgstr "सक्रिय एक्सट्रूडर के रूप में सेट करें"
+
+
+#: /home/trin/Gedeeld/Projects/Cura/resources/qml/Menus/SettingsMenu.qml:50
+msgctxt "@action:inmenu"
+msgid "Enable Extruder"
+msgstr "एक्सट्रूडर सक्षम करें"
+
+
+#: /home/trin/Gedeeld/Projects/Cura/resources/qml/Menus/SettingsMenu.qml:57
+msgctxt "@action:inmenu"
+msgid "Disable Extruder"
+msgstr "एक्सट्रूडर अक्षम करें"
+
+
+#: /home/trin/Gedeeld/Projects/Cura/resources/qml/Menus/SettingVisibilityPresetsMenu.qml:16
+msgctxt "@action:inmenu"
+msgid "Visible Settings"
+msgstr "दृश्यमान सेटिंग्स"
+
+
+#: /home/trin/Gedeeld/Projects/Cura/resources/qml/Menus/SettingVisibilityPresetsMenu.qml:45
+msgctxt "@action:inmenu"
+msgid "Collapse All Categories"
+msgstr "सभी श्रेणियाँ संक्षिप्त करें"
+
+
+#: /home/trin/Gedeeld/Projects/Cura/resources/qml/Menus/SettingVisibilityPresetsMenu.qml:54
+msgctxt "@action:inmenu"
+msgid "Manage Setting Visibility..."
+msgstr "सेटिंग की दृश्यता प्रबंधित करें..."
+
+
+#: /home/trin/Gedeeld/Projects/Cura/resources/qml/Menus/ViewMenu.qml:19
+msgctxt "@action:inmenu menubar:view"
+msgid "&Camera position"
+msgstr "&Camera पोजीशन"
+
+
+#: /home/trin/Gedeeld/Projects/Cura/resources/qml/Menus/ViewMenu.qml:45
+msgctxt "@action:inmenu menubar:view"
+msgid "Camera view"
+msgstr "कैमरा व्यूव"
+
+
+#: /home/trin/Gedeeld/Projects/Cura/resources/qml/Menus/ViewMenu.qml:48
+msgctxt "@action:inmenu menubar:view"
+msgid "Perspective"
+msgstr "दृष्टिकोण"
+
+
+#: /home/trin/Gedeeld/Projects/Cura/resources/qml/Menus/ViewMenu.qml:60
+msgctxt "@action:inmenu menubar:view"
+msgid "Orthographic"
+msgstr "ऑर्थोग्राफ़िक"
+
+
+#: /home/trin/Gedeeld/Projects/Cura/resources/qml/Menus/ViewMenu.qml:81
+msgctxt "@action:inmenu menubar:view"
+msgid "&Build plate"
+msgstr "&Build प्लेट"
```

Comment:
This one is partially translated? Normally I'd accept this since I know that Hindi uses a lot of English intermixed, but really build plate is just a single noun conceptually, so it'd be a bit weird to partially translate this?